javascript/chronometer.js

```diff
@@ -1,26 +1,43 @@
 class Chronometer {
+
   constructor() {
-    // ... your code goes here
-  }
+    this.currentTime = 0
+    this.intervalId = 0
+
+  };
+
   startClick(callback) {
-    // ... your code goes here
-  }
+    console.log("startClick")
+    this.intervalId = setInterval(() => { // arrow function is the callback
+      this.currentTime++;
+      if (callback) callback();
+    }, 1000);
+
+  }
+
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60); //61 seconds, 122 seconds
+
   }
   getSeconds() {
-    // ... your code goes here
+    return (this.currentTime % 60);
+
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+  twoDigitsNumber(num) {
+    return num.toString().padStart(2, '0');
+    // return this.currentTime.toString().padStart(2, '0');
   }
   stopClick() {
-    // ... your code goes here
+    clearInterval(this.intervalId);
   }
   resetClick() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
   splitClick() {
-    // ... your code goes here
+    let min = this.twoDigitsNumber(this.getMinutes());
+    let sec = this.twoDigitsNumber(this.getSeconds());
+    console.log("min and sec " + min + sec);
+    return `${min}:${sec}`;
+    //    return `${this.twoDigitsNumber(this.getMinutes)}\:${this.twoDigitsNumber(this.getSeconds)}`;
```

Comment:
This is fine, I just want to point out that you should remove `console.logs` and commented out code once you're finished.

It will make your code easier to read and minimal